### DISCUSSION TITLE
support WT32-SC01+ (Plus)

### DISCUSSION THREAD

**Discussion by florianru1984** [arendst/Tasmota#17858] [Q&A] [answered]
Hello, is there support for this device? I tried several Tasmota images but i cant activate the display.

[https://www.alibaba.com/product-detail/Good-heat-dissipation-IPS-LCD-Color_1600148110318.html?spm=a2700.pccps_detail.0.0.657f13a0AXVzSC](url)  

[https://doc.smartpanle.com/_static/espboard/ESP32S3/ZX3D50CE02S-USRC-4832/ZX3D50CE02S-USRC-4832.pdf](url)

**Comment by sfromis** [comment]
Should be supported. I'd expect chances of it working with
https://github.com/tasmota/install/blob/main/firmware/unofficial/tasmota32s3-lvgl.factory.bin or maybe
https://github.com/tasmota/install/blob/main/firmware/unofficial/tasmota32s3cdc-qio_opi.factory.bin
as long as you configure the relevant template
https://templates.blakadder.com/wireless_tag_WT32-SC01.html
and a uDisplay setup
https://tasmota.github.io/docs/Displays/#universal-display-driver
including this display descriptor
https://github.com/arendst/Tasmota/blob/development/tasmota/displaydesc/WT32_SC01.ini

**Comment by Jason2866** [comment]
Mhh, but 8080 mode so it will not work. The above is SPI interface.
![7A3F88BD-3A82-48F0-B1B6-A0777F7514C7](https://user-images.githubusercontent.com/24528715/216390684-5a97a962-0391-454f-9534-f78abdafd4d4.jpeg)

**Comment by Jason2866** [comment]
A cheap nice Tasmota supported display is this one https://github.com/arendst/Tasmota/pull/17414

**Comment by DieMechanikerin** [comment, marked as answer]
With the following template and display.ini it works for me. However, I have ghost touches on the middle of the y-axis :D

`{"NAME":"WT32-SC01-Plus","GPIO":[0,1,1,1,0,0,0,1,1,1,1376,1,5408,1,1,1,1,1,1,1,1,1,0,0,0,0,0,6210,0,0,0,0,1,1,0,0,0,0],"FLAG":0,"BASE":1}`

display.ini
:H,ST7796U,480,320,16,PAR,8,4,-1,0,47,-1,45,9,46,3,8,18,17,16,15,20
:I
01,A0
11,A0
3A,81,55
36,81,00
21,80
13,80
29,A0
:o,28
:O,29
:A,2A,2B,2C
:R,36
:0,28,00,00,01
:1,88,00,00,02
:2,E8,00,00,03
:3,48,00,00,00
:i,20,21
:TI1,38,5,6
#

**Reply by TimmyOtool** [reply]
wich firmware are you using?

**Reply by DieMechanikerin** [reply]
tasmota32-lvgl.bin

**Comment by loudjoudi** [comment]
Hi  did you get the touchscreen running with this config?
